js/src/pages/price-benchmark/banner/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { Button, Card, CardBody } from '@wordpress/components';
+import { createInterpolateElement, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { Pill } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import { usePreferences } from '~/hooks/usePreferences';
+import bannerImage from '~/images/price-benchmark/price-benchmark-banner-icon.svg';
+import './index.scss';
+
+const BANNER_DISMISSED_KEY = 'price-benchmark-banner-dismissed';
+
+const Banner = () => {
+	const [ visible, setIsVisible ] = useState( true );
```

Comment:
@ankitrox Technically we should not have a state to display/hide the banner. This should come from the preferences. I.e whenever the user clicks "Dismiss", their preferences gets updated and the selector returns the updated value.